core/src/main/java/jenkins/util/io/CompositeIOException.java

```diff
@@ -30,19 +30,52 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.io.IOException;
 import java.io.UncheckedIOException;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
 @Restricted(NoExternalUse.class)
 public class CompositeIOException extends IOException {
+    private static final long serialVersionUID = 121943141387608148L;
```

Comment:
Not strictly necessary, but seemed better to be safe than sorry in case someone makes a change that affects the auto-computed `serialVersionUID` in the future.